packages/turf-kinks/index.ts

```diff
@@ -40,114 +41,16 @@ export default function kinks<T extends LineString | MultiLineString | Polygon |
         type: "FeatureCollection",
         features: [],
     };
-    if (featureIn.type === "Feature") {
-        feature = featureIn.geometry;
-    } else {
-        feature = featureIn;
-    }
-    if (feature.type === "LineString") {
-        coordinates = [feature.coordinates];
-    } else if (feature.type === "MultiLineString") {
-        coordinates = feature.coordinates;
-    } else if (feature.type === "MultiPolygon") {
-        coordinates = [].concat.apply([], feature.coordinates);
-    } else if (feature.type === "Polygon") {
-        coordinates = feature.coordinates;
-    } else {
+
+    // @ts-ignore
+    if (featureIn.type === "Point" || featureIn.type === 'MultiPoint') {
```

Comment:
Should this also check for `Feature<Point>` and `Feature<MultiPoint>`?

Reply:
This might help resolve the above :)